contracts/stake-cw20-external-rewards/NOTICE

```diff
@@ -0,0 +1,13 @@
+Copyright 2021 Ben <ben2x4@tutanota.com>
```

Comment:
Do we need a NOTICE? I notice (no pun intended) none of our other contracts have them

Reply:
always have included in my contracts :shrug: